dataactbroker/handlers/accountHandler.py

```diff
@@ -302,17 +309,11 @@ def ThreadedFunction (username="", title="", cgac_code="", user_email="" , link=
                 filter(CGAC.cgac_code == cgac_code).\
                 one_or_none()
             agency_name = "Unknown" if agency_name is None else agency_name
-            for user in getUsersByType("website_admin"):
+            for user in sess.query(User).filter_by(permission_type_id=PERMISSION_TYPE_DICT['website_admin']).all():
```

Comment:
NBD, but you don't need to use all() here -- if you iterate over a query, you'll let sqlalchemy handle the paging and won't need to instantiate the whole list in memory.